README.md

```diff
@@ -7,6 +7,15 @@ This is the code for the website at bmoreonrails.org. If you'd like to help out,
 
 * Install ruby 2.0
 * `bundle install`
+* `touch config/database.yml`
+* Add the code below to your database.yml
+```
+  development:
+  adapter:  postgresql
```

Comment:
A developer might choose to run locally with sqlite rather than postgres. This is most likely why database.yml was omitted; the db implementation is up to the dev.

But I agree, to get started quickly, this is a good addition. And Postgres is a popular choice as well.